tests/sai_qualify/conftest.py

```diff
@@ -290,7 +295,8 @@ def revert_sai_test_container(duthost, creds, container_name, request):
         __restore_default_syncd(duthost, creds)
     else:
         # Prepare warmboot
-        if request.config.option.enable_warmboot_test:
+        if (request.config.option.enable_t0_warmboot_test or
```

Comment:
how about file [cases_warmreboot.py](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/sai_qualify/cases_warmreboot.py) and [test_warm_reboot.py](https://github.com/sonic-net/sonic-mgmt/blob/master/tests/sai_qualify/test_warm_reboot.py)
If there any chance that the code can use those files?